cmd/renovate/renovate-local.sh

```diff
@@ -2,17 +2,47 @@
 
 set -euo pipefail
 
-# Ensure these environment variables are set before running the script:
-# export GITHUB_TOKEN="your_github_token"
+# Ensure Github CLI is installed before running this script
+if [ ! "$(gh auth status)" ]; then
```

Comment:
Not sure about this suggestion, but what do you think about having a dedicated check for the `gh` CLI tool to exist?